### PR TITLE
System dashboards add filters

### LINKED ISSUE #4936: [System] Dashboards do not prefilter on dataset values
Looking at the system dashboards, it seems most visualisations do not prefilter on the `data_stream.dataset` value but instead query all of the `metrics-*` indices.

This is an issue as it is much less performant!

### PR DESCRIPTION
**Pull request by pchila**
## What does this PR do?

Added `dataset` filters for system metrics dashboards

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #4936

### CODE REVIEW

**Review by jlind23 — COMMENTED** [review]
Looks good to me. @leehinman @fearful-symmetry can I have 👀 on this?

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
                 "highlightAll": true,
                 "query": {
                     "language": "kuery",
-                    "query": ""
+                    "query": "host.name:\"docker-fleet-agent\""
```
Comment (leehinman) [inline packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:11]: what is this query for?  I don't remember the system dashboard having a query for docker-fleet-agent specifically.

**Review by leehinman — COMMENTED** [review]
Overall looks good.  Just one question on a query for docker-fleet-agent.

File: packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json (hunk 1 above)
Comment (pchila) [reply packages/system/kibana/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8.json:11]: You are right, there should not be a filter like that... I probably slipped up while modifying the dashboards... I will check it

**Review by jlind23 — APPROVED** [review]

**Pull request merged**